### ISSUE TITLE
Tailwind CDN not loading on Stackblitz

### ISSUE THREAD

**Issue by deadcoder0904** [tailwindlabs/tailwindcss#7169]
**What version of Tailwind CSS are you using?**

Probably the latest on CDN

**What build tool (or framework if it abstracts the build tool) are you using?**

CDN

**What version of Node.js are you using?**

v12.0.0

**What browser are you using?**

Chrome

**What operating system are you using?**

macOS

**Reproduction URL**

https://stackblitz.com/edit/tiptap-valtio?file=pages/_document.tsx

**Describe your issue**

See `pages/_document.tsx` & uncomment `script` tag pointing to Tailwind CDN. It doesn't work.

Ideally, I want to have Tailwind CDN so I can quickly style apps & instead of using `postcss` & following all the 3-4 steps, I like using CDN.

This doesn't seem like CORS. It gives the following warning in console:

> cdn.tailwindcss.com should not be used in production. To use Tailwind CSS in production, install it as a PostCSS plugin or use the Tailwind CLI: https://tailwindcss.com/docs/installation

Try uncommenting & reloading the browser window to see the error.

**Comment by reinink** [comment]
Hey!

That warning is expected — it's explaining that the Tailwind CDN should not be used in production. It's only intended for development purposes. This is mentioned [in our documentation](https://tailwindcss.com/docs/installation/play-cdn) as well:

> Use the Play CDN to try Tailwind right in the browser without any build step. The Play CDN is designed for development purposes only, and is not the best choice for production.

No issues with CORS —it's loading fine on Stackblitz. I uncommented the import, and added a Tailwind class (`bg-red-200`) to the body, and that worked without issue:

<img width="1389" alt="Screen Shot 2022-01-22 at 9 44 47 AM" src="https://user-images.githubusercontent.com/882133/150643180-cfb7ca3d-00b5-4a31-baa4-3400931b0040.png">

TLDR; Use either the Tailwind CLI or PostCSS integration for a production application 👍

**Comment by deadcoder0904** [comment]
Yes, I don't use Stackblitz for prod but bugs or small repro but idk how it works for you & not for me. Weird as hell :(

**Comment by zhe-he** [comment]
@reinink 
Is there a configuration to turn off this warning, such as calling it like this: `<script src="https://cdn.tailwindcss.com" playsinline></script>`?

I used AI to generate the HTML for me and employed Tailwind CSS for layout, so it must import the full CSS on the page rather than bundling it...
